cluster/gce/gci/configure.sh

```diff
@@ -769,7 +891,7 @@ set-broken-motd
 KUBE_HOME="/home/kubernetes"
 KUBE_BIN="${KUBE_HOME}/bin"
 
-ensure-docker
+ensure-container-runtime
```

Comment:
please verify this does run on minion nodes.

Reply:
yes, both minion nodes and master call this "configure.sh" for installation service.